domain/cloud/state/state.go

```diff
@@ -294,11 +294,11 @@ ON CONFLICT(region_uuid, key) DO UPDATE
 
 func loadClouds(ctx context.Context, tx *sql.Tx, name string) ([]cloud.Cloud, error) {
 	// First load the basic cloud info and auth types.
+	// TODO: needs controller cloud
```

Comment:
If "is_controller_cloud" is not set, bootstrap on k8s will break. 
Did we really want this change in this PR? We can't leave stuff broken - we should do the fix in a separate PR where all the relevant work can be completed.

Reply:
I'll work on this. But we can ask this question we just need to wire it up a little better.

Reply:
This is the right move though. The relational algebra has no way to constrain having is_controller_cloud=true for a cloud that is not used by the controller.

So always answering that question by SQL is correct.